README.md

```diff
@@ -219,12 +204,6 @@ Example:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
```

Comment:
- [ ] Kindly don't forget that as a general requirement you should always describe 1 - 3 features you will add to your project :+1: because it informs to users of what to expect in the future and helps manage their expectations.